pkg/observability/modules/suseObservability.ts

```diff
@@ -78,26 +78,57 @@ export function isCrdLoaded(store: any): boolean {
   return loaded;
 }
 
+export async function isSuseObservabilityRepoPresent(store: any): Promise<Boolean> {
+  console.log('Checking if Observability Repo is present');
+  const repos = await store.dispatch('management/findAll', { type: 'catalog.cattle.io.clusterrepo' });
+
+  console.log('Checking if Observability Repo is present', repos);
+  if (!repos || isEmpty(repos)) {
+    return false;
+  }
+
+  const suseObservabilityRepo = repos.find(
+    (s: any) => s.metadata.name === 'stackstate' || s.metadata.name === 'suse-observability'
+  );
+
+  console.log('Checking if Observability Repo is present', suseObservabilityRepo);
+
+  return suseObservabilityRepo && !isEmpty(suseObservabilityRepo);
```

Comment:
if `suseObservabilityRepo` present then `isEmpty(suseObservabilityRepo)` always returns false

so it can use `repos.some` instead of `repos.find`